### PR TITLE
INS Taqot 1007

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New repository.

### CODE REVIEW

File: new/INS1007Taqot_Maryam.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        <geo>14.172079 39.564739</geo>
+                        <height unit="m">2635.0</height>
+                    </location>
+                    <note>
+                        <date type="foundation" notBefore="0200" notAfter="1500"/>
+                        <ab type="history">Dabr.</ab>
+                        <ab type="tabot">
+                            <measure quantity="3">3</measure>  3 tābots,                                
+                                  <persName ref="PRS6819Mary" type="tabot">Mary</persName>,  
+                            <persName ref="PRS9684Uriel" type="tabot">ʿUraʾel</persName>,
```
Comment (thea-m) [inline new/INS1007Taqot_Maryam.xml:73]: ʿUrāʾel

File: new/INS1007Taqot_Maryam.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                            <persName ref="PRS1284Abrehaa" type="tabot">Abrǝhā and ʾAṣbǝḥa</persName>.                                                               
+                        </ab>
+                                                <ab type="description">The church of St. Mary Zion stands on the foundation dating at least to the Aksumite time,
```
Comment (thea-m) [inline new/INS1007Taqot_Maryam.xml:76]: St

File: new/INS1007Taqot_Maryam.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
```
Comment (thea-m) [inline new/INS1007Taqot_Maryam.xml:37]: Tigrinya to be defined here

**Review by thea-m — CHANGES_REQUESTED** [review]
Nice :)

**Review by thea-m — APPROVED** [review]

File: new/INS1007Taqot_Maryam.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-03">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">                    
+                    <placeName xml:id="n1" xml:lang="gez">ጣቖት፡ ደብረ፡ ጽዮን፡ ማርያም፡</placeName>
```
Comment (eu-genia) [inline new/INS1007Taqot_Maryam.xml:49]: gez or ti? here and everywhere

File: new/INS1007Taqot_Maryam.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1007Taqot_Maryam" xml:lang="en" type="ins">
```
Comment (eu-genia) [inline new/INS1007Taqot_Maryam.xml:4]: what is done is done, but we should avoid hyphens and underscores in xml:id and filenames, as is written in the Guidelines

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS1007Taqot_Maryam.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/INS1007Taqot_Maryam.xml:49]: both languages, strictly speaking.

File: new/INS1007Taqot_Maryam.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/INS1007Taqot_Maryam.xml:4]: It seems that hyphen was generated by the system, could it be? Did I had to delete it manually?

Comment (eu-genia) [reply new/INS1007Taqot_Maryam.xml:4]: You must have entered Taqot Maryam (with a space) in the form, we cannot use spaces, these become the underscore, but we should not use underscores either. You should have entered TaqotMaryam as one word then there would be no error

File: new/INS1007Taqot_Maryam.xml (hunk 4 above)
Comment (eu-genia) [reply new/INS1007Taqot_Maryam.xml:49]: if we say we have ቖ in gez then OK

Comment (DenisNosnitsin1970) [reply new/INS1007Taqot_Maryam.xml:49]: Däbrä Seyon is Geez construction. Taqot is a Tigrigna word.

File: new/INS1007Taqot_Maryam.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/INS1007Taqot_Maryam.xml:4]: Ok, probably it is my mistake, then.

Comment (eu-genia) [reply new/INS1007Taqot_Maryam.xml:4]: I have added a prompt to the form

**Pull request merged**